deltachat-ffi/deltachat.h

```diff
@@ -686,6 +686,22 @@ int             dc_get_connectivity          (dc_context_t* context);
 char*           dc_get_connectivity_html     (dc_context_t* context);
 
 
+#define DC_PUSH_NOT_CONNECTED 0
+#define DC_PUSH_HEARTBEAT     1
```

Comment:
we need sth. as DC_PUSH_CONNECTED (2) or so

in the UI, we show the notification state meanwhile very picky (battery, data, whatnot)  in the connectivity view.

if sth. is not working, we point users there to see if everything's green.

therefore, UI should know that "real notifications" are used - that would be only only case where we would show the line "green" as "connected" :)

![IMG_5249](https://github.com/deltachat/deltachat-core-rust/assets/9800740/24977a20-67ad-43bf-9371-b27746103dc0)

Reply:
thanks! i adapted https://github.com/deltachat/deltachat-ios/pull/2080

![IMG_5249](https://github.com/deltachat/deltachat-core-rust/assets/9800740/1ddc470f-820d-4466-bef2-56974e863cab)